### DISCUSSION TITLE
sync error: Failed to receive header: The TLS connection was non-properly terminated. Sync failed.  Could not connect to the Taskserver

### DISCUSSION THREAD

**Discussion by djotaku** [coddingtonbear/inthe.am#423] [Q&A] [answered]
Ever since yesterday I'm getting the following error when i do task sync:

Failed to receive header: The TLS connection was non-properly terminated.
Sync failed.  Could not connect to the Taskserver.

Not sure if this is something on my end, but after a bunch of googling, I couldn't find this exact issue on here, so I'm not sure what to do.

**Comment by coddingtonbear** [comment, marked as answer]
This is almost certainly just that your certificate has expired.  You can download a new one from the website.

**Reply by djotaku** [reply]
that fixed it. Thank you.
